### ISSUE TITLE
The namespace of the generated press handler is 'ui5.cc.v4.samplev4excelupload'

### ISSUE THREAD

**Issue by miyasuta** [marianfoo/generator-excelUpload#1]
## Easy-Ui5 version:

> Run `yo easy-ui5 --plugins` to print all relevant version strings

```
Node.js: v18.14.2
yeoman-environment: 3.10.0
generator-easy-ui5: 3.5.0
home: C:\Users\mio-f\AppData\Roaming\npm\node_modules\generator-easy-ui5
pluginsHome: C:\Users\mio-f\AppData\Roaming\npm\node_modules\generator-easy-ui5\plugin-generators

Available generators:
  - ui5-project: 0.2.1
  - ui5-ts-app: 1.0.1
 ```

## OS/version:
Windows/11 Home

## Browser/version (+device/version):
Google Chrome  109.0.5414.121

## Steps to reproduce the problem:

1. Execute `yo ui5-excelupload` on my project

## What is the expected result?
It sets the id of my own project to the press handler in manifest.json

```
              "content": {
                "header": {
                  "actions": {
                    "excelUpload": {
                      "id": "excelUploadButton",
                      "text": "Excel Upload",
                      "enabled": "{ui>/isEditable}",
                      "press": "mynamespace.myproject.ext.ObjectPageExtController.openExcelUploadDialog",
                      "requiresSelection": false
                    }
                  }
                }
              }
```

## What happens instead?
It sets "ui5.cc.v4.samplev4excelupload" to the press handler.

```
              "content": {
                "header": {
                  "actions": {
                    "excelUpload": {
                      "id": "excelUploadButton",
                      "text": "Excel Upload",
                      "enabled": "{ui>/isEditable}",
                      "press": "ui5.cc.v4.samplev4excelupload.ext.ObjectPageExtController.openExcelUploadDialog",
                      "requiresSelection": false
                    }
                  }
                }
              }
```

Ideally, the setting from the target project should be used.

**Comment by marianfoo** [comment]
Thank you for your report, you´re right, i missed this
Fixed with 0.2.2

**Comment by miyasuta** [comment]
Thank you for your prompt action!
